service/topology_state_machine.hh

```diff
@@ -110,6 +110,7 @@ struct topology {
         write_both_read_old,
         write_both_read_new,
         tablet_migration,
+        tablet_split,
```

Comment:
This is really just the finalization, yes? tablet split happens in parallel with migration.

Please update docs/dev/topology-over-raft.md too.

Reply:
> This is really just the finalization, yes? tablet split happens in parallel with migration.

Correct.

> 
> Please update docs/dev/topology-over-raft.md too.

Will do.

Reply:
Indeed, I think it's better to have a more explicit name to avoid confusing others.